pkg/cmd/provisioning/ibmcloud/create_service_id_test.go

```diff
@@ -644,7 +644,7 @@ func mockListResourceGroups(client *mockibmcloud.MockClient, resourceGroupExist,
 	client.EXPECT().ListResourceGroups(gomock.Any()).Return(list, nil, err).Times(1)
 }
 
-func mockListServiceID(client *mockibmcloud.MockClient, namePrefix string, count int, fail bool) {
+func mockListServiceIDTimes(client *mockibmcloud.MockClient, namePrefix string, count int, fail bool, times int) {
```

Comment:
The var names 'count' and 'times' are a bit close in meaning. Can we rename 'count' to something like 'numSvcIDs' (or just something that makes the difference between 'count' and 'times' more immediately clear?

Reply:
done, renamed to `numSvcIDs`